spec/tasks/delete_goals_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'rake'
+WithinAWeek::Application.load_tasks
```

Comment:
Good work loading the Rake tasks here and apologies for forgetting to reference the config that can also do this another way: https://github.com/eliotsykes/rspec-rails-examples/blob/master/spec/support/tasks.rb